src/witan/models/dem/ccm/fert/fertility.clj

```diff
@@ -145,11 +145,11 @@
    gather function in witan.datasets similar to gather in tidyR"
   [births-by-mf]
   (let [births-f (-> births-by-mf
-                     (ds/add-column :sex (repeat "F"))
+                     (ds/add-column :sex (repeat 1 "F"))
```

Comment:
So this was an interesting, bug, adding a length calculator to `births-by-mf` (which still returns 1), breaks the code when it is added as the first argument to `repeat`.
i.e. births-by-mf-n (wds/row-count data)
      births-f (-> births-by-mf
-                     (ds/add-column :sex (repeat  births-by-mf-n "F"))